src/rest_vol_attr.c

```diff
@@ -114,6 +116,20 @@ RV_attr_create(void *obj, const H5VL_loc_params_t *loc_params, const char *attr_
     new_attribute->domain = parent->domain;
     parent->domain->u.file.ref_count++;
 
+    new_attribute->handle_path = NULL;
+
+    if (RV_set_object_handle_path(attr_name, parent->handle_path, &new_attribute->handle_path) < 0)
```

Comment:
Since the documentation for `H5Iget_name` says that it returns the name of the object to which the attribute is attached, I don't think you want to add the attribute's name into `handle_path` here. In fact, you should just be able to return `parent->handle_path` when `H5Iget_name` is called on an attribute.

Reply:
This case is handled [here](https://github.com/HDFGroup/vol-rest/pull/41/files#diff-de2be4672626f5b68efea83e058e0d4b07beb9fd876ed4703032102ed3135e2eR308) in the logic for `H5Iget_name` itself. I think having the attribute store its name in its path is more consistent, and more flexible in case any other API functions work differently.